packages/vc-verification/src/Ipfs-credential-resolver.ts

```diff
@@ -117,7 +117,12 @@ export class IpfsCredentialResolver implements CredentialResolver {
     return eip191JwtProps.every((p) => claimProps.includes(p));
   }
 
-  private async eip191JwtOf(did: string): Promise<RoleEIP191JWT[]> {
+  /**
+   * Fetches all the Role eip191Jwts based on the subject DID
+   * @param did subject DID
+   * @returns RoleEIP191JWT list
+   */
+  async eip191JwtOf(did: string): Promise<RoleEIP191JWT[]> {
```

Comment:
Maybe consider adding an `s` here to indicate a plural return (as credential**s**Of does)
```suggestion
  async eip191JwtsOf(did: string): Promise<RoleEIP191JWT[]> {
```